pmm-app/package.json

```diff
@@ -58,7 +58,7 @@
     "@types/react-dom": "^16.9.6",
     "@types/react-router-dom": "^5.1.5",
     "babel-jest": "^24.9.0",
-    "codeceptjs": "^2.6.1",
+    "codeceptjs": "^2.6.5",
```

Comment:
When upgrading packages related to the test suite, we should also try to keep companion packages upgraded. For example, we could potentially upgrade:
- codeceptjs-resemblehelper to v1.9.1
- playwright to v1.0.2
- playwright-video to v1.0.0
- codecov v3.7.0

This can be done one by one, if the tests pass - good, if they don't - rollback to the current version.
Actually, there may breaking changes for playwright, so proceed with care :)

Reply:
Just discovered that playwright 1.x is supported since v2.6.4.

Reply:
moved version back to 2.6.1, looks like we might need to update it along with updating docker image version